src/runtime/pkg/qemu/qemu.go

```diff
@@ -244,6 +244,7 @@ const (
 )
 
 // Object is a qemu object representation.
+// nolint:govet
```

Comment:
About this one and a number of other changes like this, two comments:

1. Having everything disabled feels like we're leaving this behind so someone else can solve it, which is not exactly what I'd expect.  @devimc, do you have any opinions on this?
2. In all other parts of the code we're using `// nolint: govet`, it's a nitpick but helps with consistency.

About 1, if we're taking this path of just ignoring those, wouldn't be better to not bother about those at all and disable it on https://github.com/kata-containers/tests/blob/f3724e6c2bf17f15943bd98b4e7f1588bb0c0319/.ci/.golangci.yml#L36 ?  If we care about those, I'd advocate for having this properly fixed *before* we merge this.

Reply:
let's move forward and merger this, I've filed https://github.com/kata-containers/kata-containers/issues/2456 to track this and fix this later, what do you think ?

Reply:
Yeah, I think handling it after will be an easier approach.  Apart from anything else, I'm considering simply removing some of the structures that govet is complaining about (since they're basically intermediate forms between Kata's internal representation and qemu's representation with no real purpose).